src/lumigo_tracer/event/event_trigger.py

```diff
@@ -87,6 +102,20 @@ def _parse_http_method(event: dict):
     return _parse_http_method_v1(event)
 
 
+def _parse_load_balancer_method(event: dict):
+    result = {
+        "triggeredBy": "load_balancer",
+        "httpMethod": event.get("httpMethod", ""),
+        "resource": event.get("resource", ""),
+        "messageId": event.get("requestContext", {}).get("requestId", ""),
```

Comment:
I don't think that this ever exists..
https://docs.aws.amazon.com/lambda/latest/dg/services-alb.html